src/tactic/algebra.lean

```diff
@@ -20,6 +20,24 @@ private meta def reflect_name_list : has_reflect (list name) | ns :=
 private meta def parse_name_list (e : expr) : list name :=
 e.app_arg.get_app_args.map expr.const_name
 
+/-- The `ancestor` attributes is used to record the names of structures which appear in the
+extends clause of a `structure` or `class` declared with `old_structure_cmd` set to true.
+
+As an example::
+```
+set_option old_structure_cmd true
+
+structure base_one := (one : ℕ)
+
+structure base_two (α : Type*) := (two : ℕ)
+
+@[ancestor base_one base_two]
+structure bar extends base_one, base_two α
+```
+
+The list of ancestors should be in the order they appear in the `extends` clause, and should
+contain only the names of the ancestor structures, without any arguments.
+-/
 @[user_attribute]
 private meta def ancestor_attr : user_attribute unit (list name) :=
```

Comment:
```suggestion
meta def ancestor_attr : user_attribute unit (list name) :=
```
The attribute doesn't show up in the web docs if it's `private`: https://leanprover-community.github.io/mathlib_docs/tactic/algebra.html